sbin/start_sp_magi.sh

```diff
@@ -53,8 +51,5 @@ CMD="$MAGI \
     $DEVNET_FLAGS \
     $SEQUENCER_FLAGS"
 
-# TODO: wait for config.json to be generated
-sleep 15
```

Comment:
this will cause the docker compose to fail.
Currently trying to use like this [this](https://docs.docker.com/compose/compose-file/05-services/#depends_on) to start containers in the correct order without waiting in the scripts.